cpp_guidelines/.clang-format

```diff
@@ -0,0 +1,107 @@
+---
+Language: Cpp
+Standard: c++17
+DisableFormat:   false
+AccessModifierOffset: -4
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: true
+AlignConsecutiveBitFields: true
```

Comment:
VOTE

current | false |
--------|-------------|
 0      |   1         |

Alignment always results into the issue that one line change can trigger re-alignment of other lines and will increase the diff.

Reply:
current | None
-- | --
0 | 2

Reply:
current | None
-- | --
0 | 3

Reply:
current | None |
----------|---------|
1 | 3

Reply:
I think for bitfields the alignment issue is less of a problem, because the scope is more limited in which they occur.

current | None
-- | --
2 | 3

Reply:
Voting closed, the winner is `false`. However, this vote won't be implemented until clang-format 11 is released (clang-format 10 doesn't support this option).